engine/engine/src/wscript

```diff
@@ -132,9 +132,13 @@ def build(bld):
     if bld.env['PLATFORM'] in ('x86_64-ps4', 'x86_64-ps5'):
         additional_libs.append('APP')
 
+    platform_lib = ['PLATFORM']
+    if not platform_supports_feature(bld.env.PLATFORM, "opengl", None):
+        platform_lib = ['PLATFORM_NULL']
+
```

Comment:
Currently PLATFORM implies opengl, which is a bit odd imho.
Should be able to configure it better, not sure how yet though...

Reply:
Hm, but it doesn't _really_ imply opengl only, you pass in an enum to select which context to create, but right now opengl is the only adapter that can be used with the vanilla platform (vulkan has to use platform_vulkan) but when we have the directx support it can be used for both.

Reply:
On that note, it's a more annoying issue that "graphics.lib" implies opengl whereas the others don't :)

Reply:
Yeah, I think we can take another stab at consolidating the features to make it more "clean" in how we configure it.